doglist.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>>Adopt A Dog</title>
+
+  <link rel="stylesheet" type="text/css" href="./styles/doglist.css">
+
+</head>
+
+<body>
+  <header>
+    <img src="./images/adoptadog-logo.svg" alt="adopt a dog logo">
+    <h1>Seattle's Premiere dog adoption agency</h1>
+    <img src="./images/cart-icon.svg" alt="shopping cart icon">
+    <p>check out</p>
+  </header>
+
+<main>
+  <nav class="col span_3_of_12">
+    <ul class="menunavigation">
+      <li class="menunavigationitem"><a href="./index.html">Home</a></li>
+      <li class="menunavigationitem"><a href="./doglist.html">Dogs</a></li>
+      <li class="menunavigationitem"><a href="./blog.html">Blog</a></li>
+    </ul>
+  </nav>
+
+<h2>Here are all of our Dogs!</h2>
+<section>
+  <img src="./images/murphy-card.jpg" alt="picture of dog named murphy">
```

Comment:
Looking good, but you'll want to wrap each of these dog tiles in some sort of wrapping element. It will make layout much easier